book/src/theme-gallery/index.md

```diff
@@ -106,3 +107,28 @@ It features a dominant partition of space into a bright and a dark side.
 ```
 ![bipartite theme screenshot](./bipartite.png)
 
+## University
+_[go to top](#theme-gallery)_
+
+This theme is a simple and configurable theme.
+
+### Options
+- `university-name`: The University name
+- `color-a`: A color of your choice
+- `color-b`: A color of your choice
+- `color-c`: A color of your choice
```

Comment:
Can we find more descriptive explanations for these colors? Like, what is their function in the theme?

Reply:
Fixed.